security/providers/oidc/pom.xml

```diff
@@ -58,18 +58,6 @@
             <groupId>io.helidon.security.providers</groupId>
             <artifactId>helidon-security-providers-common</artifactId>
         </dependency>
-        <dependency>
-            <groupId>io.helidon.security.integration</groupId>
-            <artifactId>helidon-security-integration-webserver</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>io.helidon.reactive.webserver</groupId>
-            <artifactId>helidon-reactive-webserver</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>io.helidon.reactive.webserver</groupId>
-            <artifactId>helidon-reactive-webserver-cors</artifactId>
-        </dependency>
         <dependency>
```

Comment:
Either all dependencies on Níma MUST be as `provided` and `optional` with `requires static` in `module-info.java`, or Níma OIDC integration must be moved to a separate module

Reply:
Updated